pkg/agent/externalnode/external_node_controller.go

```diff
@@ -53,11 +53,11 @@ const (
 	// Disable resyncing.
 	resyncPeriod time.Duration = 0
 
-	ovsExternalIDUplinkName      = "uplink-name"
-	ovsExternalIDHostIFName      = "hostInterface-name"
-	ovsExternalIDUplinkPort      = "uplink-port"
-	ovsExternalIDEntityName      = "entity-name"
-	ovsExternalIDEntityNamespace = "entity-namespace"
+	OvsExternalIDUplinkName      = "uplink-name"
```

Comment:
No need to make `OvsExternalIDUplinkName` public since it is consumed only in this package?

Reply:
So do other ovsExternalIDXXXX

Reply:
Sure, will check and update.